<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="Bookmarks" name="listWidget">
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="viewMode">
        <enum>QListView::IconMode</enum>
       </property>
      </widget>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="openAct"/>
    <addaction name="insertAct"/>
    <addaction name="replaceAct"/>
    <addaction name="separator"/>
    <addaction name="saveAct"/>
    <addaction name="saveAsAct"/>
    <addaction name="createTIFFAct"/>
    <addaction name="separator"/>
    <addaction name="exitAct"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="selectAllAct"/>
    <addaction name="selectOddAct"/>
    <addaction name="selectEvenAct"/>
    <addaction name="separator"/>
    <addaction name="deleteAct"/>
    <addaction name="moveAct"/>
    <addaction name="separator"/>
    <addaction name="rotateCWAct"/>
    <addaction name="rotateCCWAct"/>
    <addaction name="rotate180Act"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="zoomInAct"/>
    <addaction name="zoomOutAct"/>
    <addaction name="fitToWindowAct"/>
    <addaction name="fitWidthAct"/>
    <addaction name="fit100Act"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="openAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/icn_Toolbar_simplescan.png</normaloff>:/images/assets/icn_Toolbar_simplescan.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="insertAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/icn_Toolbar_add.png</normaloff>:/images/assets/icn_Toolbar_add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Insert</string>
   </property>
  </action>
  <action name="replaceAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/icn_Toolbar_replace.png</normaloff>:/images/assets/icn_Toolbar_replace.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Replace</string>
   </property>
  </action>
  <action name="saveAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/icn_Toolbar_save.png</normaloff>:/images/assets/icn_Toolbar_save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="saveAsAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/icn_Toolbar_save.png</normaloff>:/images/assets/icn_Toolbar_save.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As</string>
   </property>
  </action>
  <action name="createTIFFAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/icn_Toolbar_multifeed.png</normaloff>:/images/assets/icn_Toolbar_multifeed.png</iconset>
   </property>
   <property name="text">
    <string>Create &amp;TIFF</string>
   </property>
   <property name="toolTip">
    <string>Create TIFF</string>
   </property>
  </action>
  <action name="exitAct">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="selectAllAct">
   <property name="text">
    <string>Select &amp;All</string>
   </property>
  </action>
  <action name="selectOddAct">
   <property name="text">
    <string>Select &amp;Odd</string>
   </property>
  </action>
  <action name="selectEvenAct">
   <property name="text">
    <string>Select &amp;Even</string>
   </property>
  </action>
  <action name="deleteAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/icn_Toolbar_delete.png</normaloff>:/images/assets/icn_Toolbar_delete.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
  </action>
  <action name="moveAct">
   <property name="text">
    <string>&amp;Move</string>
   </property>
  </action>
  <action name="rotateCWAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/icn_Toolbar_rotate90.png</normaloff>:/images/assets/icn_Toolbar_rotate90.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Rotate CW</string>
   </property>
  </action>
  <action name="rotateCCWAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/icn_Toolbar_rotate-90.png</normaloff>:/images/assets/icn_Toolbar_rotate-90.png</iconset>
   </property>
   <property name="text">
    <string>Rotate CCW</string>
   </property>
   <property name="toolTip">
    <string>Rotate CCW</string>
   </property>
  </action>
  <action name="rotate180Act">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/icn_Toolbar_rotate180.png</normaloff>:/images/assets/icn_Toolbar_rotate180.png</iconset>
   </property>
   <property name="text">
    <string>Rotate 180</string>
   </property>
  </action>
  <action name="zoomInAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/icn_Toolbar_zoom_in.png</normaloff>:/images/assets/icn_Toolbar_zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;In</string>
   </property>
  </action>
  <action name="zoomOutAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/icn_Toolbar_zoom_out.png</normaloff>:/images/assets/icn_Toolbar_zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;Out</string>
   </property>
  </action>
  <action name="fitToWindowAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/icn_Toolbar_fit2window.png</normaloff>:/images/assets/icn_Toolbar_fit2window.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Fit to Window</string>
   </property>
   <property name="toolTip">
    <string>Fit To Window</string>
   </property>
  </action>
  <action name="fitWidthAct">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/icn_Toolbar_fit2width.png</normaloff>:/images/assets/icn_Toolbar_fit2width.png</iconset>
   </property>
   <property name="text">
    <string>Fit &amp;Width</string>
   </property>
  </action>
  <action name="fit100Act">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/images/assets/icn_Toolbar_100.png</normaloff>:/images/assets/icn_Toolbar_100.png</iconset>
   </property>
   <property name="text">
    <string>Fit &amp;100</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Bookmarks</class>
   <extends>QListWidget</extends>
   <header>Bookmarks</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="rsrc.qrc"/>
 </resources>
 <connections/>
</ui>
